tests/ui/auto-traits/auto-traits-type-parameter

```diff
@@ -1,24 +1,25 @@
+//! Checks how type parameters interact with auto-traits like `Send` and `Sync` (implicitly),
+//! even when no explicit trait bounds are provided for `Send` or `Sync`.
```

Comment:
```suggestion
//! Checks how type parameters interact with auto-traits like `Send` and `Sync` with implicit
//! bounds
```

Or something similar to reduce the redundancy between `(implicit)` and `even when no explicit...`